### ISSUE TITLE
Keyboard nav on Google TV?

### ISSUE THREAD

**Issue by larham** [jfeinstein10/SlidingMenu#364]
On a Google TV device, I'm seeing problems with keyboard navigation on the SlidingMenu sample app, using the directional pad (d-pad) on either the content pane or the sliding menu:  

On a self-built sample app, the samples fail to respond to keyboard navigation _after_ showing the sliding menu. 

However, when I manually adb-push the sample .apk from Google Play, some samples seem to have working navigation after showing the menu.

Questions:
- has anyone tried Google TV in the past? 
  -- are there any apps that use SlidingMenu in the Google Play that is filtered for GTV?
- what ActionBarSherlock version was used to build the samples in Google Play?

steps to reproduce:
- build sample app (changed one library file's superclass to a Sherlock class)
- put sample app on a Google TV device  
- open any sample; the Attach sample is good since it has a list in the content page
  -- initially, before sliding menu is shown, nav works ok on content page
- open sliding menu by clicking "center" d-pad (click) on the menu icon
  -- note that all nav is broken thereafter--up/down/left/right have no visual effect, though "center" click can have effect; clicking 'back' button will close menu, but previously-working content list has no further response to d-pad. There is no error logging I can see; it is as though "touch mode" has taken over, or the d-pad rendered inoperative.

**Comment by jfeinstein10** [comment]
I don't know of any apps specifically for Google TV using SlidingMenu. Actually, I don't personally have one either :frowning: 
Any version of ABS can drop right in.
